core/src/main/kotlin/org/jetbrains/research/testspark/core/data/ClassType.kt

```diff
@@ -9,4 +9,7 @@ enum class ClassType(val representation: String) {
     INTERFACE("interface"),
     ABSTRACT_CLASS("abstract class"),
     CLASS("class"),
+    DATA_CLASS("data class"),
+    INLINE_VALUE_CLASS("inline value class"),
```

Comment:
What is **inline value class**?

Reply:
I implemented different structures from the classes and objects section and [inline value classes](https://kotlinlang.org/docs/inline-classes.html) is one of them